src/docGeneratorMain.ts

```diff
@@ -76,7 +76,7 @@ async function main() {
   console.log("Generate eslint description files")
   await docGenerator.downloadDocs(
     (pattern) =>
-      `${githubBaseUrl}/eslint/eslint/master/docs/rules/${pattern}.md`
```

Comment:
Here what we want is to use a stable tag instead of `master`
We have the toolVersion already that you can import with:
```ts
import { toolVersion } from "./toolMetadata"
```
and then you probably need to prepend a `v` to it